apps/tailwind-components/components/input/listbox/List.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ul
+    role="listbox"
+    ref="ul"
```

Comment:
why do you need these refs? In other components we simply use props to cascade changes down.